sdk/keyvault/azure-security-keyvault-jca/README.md

```diff
@@ -66,12 +66,46 @@ az keyvault create --resource-group <your-resource-group-name> --name <your-key-
 ```
 
 ## Key concepts
+### SSL/TLS and mTLS
+The JCA library supports SSL/TLS and mTLS (Mutual TLS) to enhance security in secure communication channels. It enables applications to securely retrieve certificates from Azure Key Vault and use them for TLS-related operations.
+
+### Jar Signer
+The JCA library provides support for Java Archive (JAR) signing, ensuring the integrity and authenticity of JAR files using certificates stored in Azure Key Vault.
 
 ## Examples
-### Server side SSL
+### Exposed Options
+The JCA library supports configuring the following options:
+* `azure.keyvault.uri`: The Azure Key Vault endpoint to retrieve certificates.
+* `azure.keyvault.aadAuthenticationUrl`: The authentication URL for Microsoft Entra ID.
+* `azure.keyvault.tenantId`: The Microsoft Entra ID tenant ID required for authentication.
+* `azure.keyvault.clientId`: The client/application ID used for authentication.
+* `azure.keyvault.clientSecret`: The client secret for authentication when using client credentials.
+* `azure.keyvault.managedIdentity`: Indicates whether Managed Identity authentication is enabled.
+* `azure.cert-path.well-known`: The path where the well-known certificate is stored.
+* `azure.cert-path.custom`: The path where the custom certificate is stored.
+* `azure.keyvault.jca.refresh-certificates-when-have-un-trust-certificate`: Indicates whether to refresh certificates when have untrusted certificate.
+* `azure.keyvault.jca.certificates-refresh-interval(-in-ms)`: The refresh interval time.
```

Comment:
Would it be clearer if it were split into two parts? 
- azure.keyvault.jca.certificates-refresh-interval
- azure.keyvault.jca.certificates-refresh-interval-in-ms